api/src/main/java/com/sanction/lightning/models/FacebookPost.java

```diff
@@ -0,0 +1,20 @@
+package com.sanction.lightning.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.restfb.types.Post;
+
+import java.util.List;
+
+public class FacebookPost {
+
```

Comment:
You can keep this class but can you just remove all the code inside? It's gonna change when you add support for getting the news feed.